src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -207,6 +207,78 @@ void M()
             EOF();
         }
 
+        [Fact]
+        public void TestInterpolatedStringWithNewLinesInExpression()
+        {
+            var text = @"$""Text with {\nnew[] {\n 1, 2, 3 \n}[2]\n}parts and new line expressions!"""
+                .Replace("\\n", System.Environment.NewLine);
```

Comment:
I don't understand the need for this. Why are we not just putting the newlines in the verbatim string?